### DISCUSSION TITLE
トップページに表示される記事の投稿日が違う

### DISCUSSION THREAD

**Discussion by toru1055** [increments/qiita-discussions#294] [要望・アイデア・不具合]
ありがたいことに私の記事「[Tech記事の検索とレコメンデーションが可能なサービスを個人開発しました！](https://qiita.com/toru1055/items/e7239c00f7c9ee96ed3f)」がトップページに掲載されたのですが、投稿日が間違って表示されています（記事詳細ページは正しく表示されている）。

実際の投稿日時： 2022年09月09日
トップページ上での投稿日時：2022年09月08日

AM6時頃に投稿したので、おそらくトップページの方がUTC基準で日付を表示しているのではと思います。

[似た投稿](https://github.com/increments/qiita-discussions/discussions/125)もありましたが、半年ほど前に修正済みとのことなので、再度ご確認いただければと思います。

## 記事詳細ページ
![スクリーンショット 2022-09-10 10 39 38](https://user-images.githubusercontent.com/5128987/189464004-580382d8-0684-4d11-a194-8d700fa83617.png)

## トップページ
![スクリーンショット 2022-09-10 10 38 09](https://user-images.githubusercontent.com/5128987/189463966-2586e253-19a3-4763-9ff2-dd863d76e991.png)

**Comment by tanaka-asami415** [comment]
@toru1055 
ご報告ありがとうございます🙏こちら調査いたします！🙇

**Comment by kato647** [comment]
@toru1055 
大変お待たせしました 🙇
ホームの記事の投稿日と記事ページの投稿日が異なる不具合を解消しました。
ホーム画面をご確認いただければと思います！
